tests/Mvp.Selections.Api.Tests/Services/LicenseServiceTests.cs

```diff
@@ -0,0 +1,223 @@
+        private readonly IUserRepository _userRepository;
+
+        public LicenseServiceTests()
+        {
+            _licenseRepository = Substitute.For<ILicenseRepository>();
+            _logger = Substitute.For<ILogger<LicenseService>>();
+            _userRepository = Substitute.For<IUserRepository>();
+            _sut = new LicenseService(_licenseRepository, _userRepository, _logger);
+
+            Fixture fixture = new Fixture();
+            fixture.Behaviors
+                .OfType<ThrowingRecursionBehavior>()
+                .ToList()
+                .ForEach(b => fixture.Behaviors.Remove(b));
+            fixture.Behaviors.Add(new OmitOnRecursionBehavior());
+        }
+
+        [Fact]
+        public async Task AssignUserAsync_WithInvalidUser_ReturnBadRequest()
+        {
+            // Arrange
+            Guid licenseId = Guid.NewGuid();
+            var request = new AssignUserToLicense
+            {
+                Email = string.Empty
+            };
+
+            // Act
+            var result = await _sut.AssignLicenseToUserAsync(request, licenseId);
+
+            // Assert
+            result.StatusCode.Should().Be(HttpStatusCode.BadRequest);
+            result.Messages.Should().Contain("Email is required");
+        }
+
+        [Fact]
+        public async Task AssignUserAsync_WithValidUserAndLicense_ReturnsSuccess()
+        {
+            // Arrange
+            var licenseId = Guid.NewGuid();
+            var email = "test@example.com";
+            var license = new Domain.License(licenseId)
+            {
+                LicenseContent = "test content",
+                ExpirationDate = DateTime.UtcNow.AddDays(30)
+            };
+
+            var user = new User(Guid.NewGuid())
+            {
+                Email = email
+            };
+
+            var request = new AssignUserToLicense
+            {
+                Email = email
+            };
+
+            _licenseRepository.GetLicenseAsync(licenseId)
+                .Returns(license);
+
+            _userRepository.GetAllAsync(email: email)
+                    .Returns(new List<User> { user });
+
+            _licenseRepository.UserHasTitleForYearAsync(user, DateTime.Now.Year)
+                .Returns(true);
+
+            _licenseRepository.AssignLicenseToUserAsync(Arg.Any<Domain.License>())
+                    .Returns(Task.CompletedTask);
+
+            _licenseRepository.SaveChangesAsync()
+                .Returns(Task.CompletedTask);
+
+            // Act
+            var result = await _sut.AssignLicenseToUserAsync(request, licenseId);
+
+            // Assert
+            result.StatusCode.Should().Be(HttpStatusCode.OK);
+            result.Result.Should().Be(license);
+
+            await _licenseRepository.Received(1)
+                .AssignLicenseToUserAsync(Arg.Is<Domain.License>(l => l.AssignedUserId == user.Id));
+        }
+
+        [Fact]
+        public async Task AssignUserAsync_WithInvalidLicenseId()
```

Comment:
Name doesn't insclude the expected result  at the end, should be `AssignUserAsync_WithInvalidLicenseId_XXXX`